src/components/TopBar.js

```diff
@@ -12,21 +12,33 @@ const topBarTextColor = 'white';
 export const styles = {
   title: {
     color: topBarTextColor,
+    display: 'flex',
   },
   icon: {
     color: topBarTextColor,
+    display:'flex',
   },
+  topBarSectionStyle: {
```

Comment:
I think calling it `somethingStyle` is a bit redundant - we know the classNames are for styling. Why not name it after the component we're styling? `toolbar` seems like a good className to me.